### PR TITLE
Cql4bonnie qdm5 3 937 facility

### PR DESCRIPTION
**Pull request by lukoktonos**
JIRA Test: https://jira.mitre.org/browse/BONNIE-968

Related PRs:
- https://github.com/projecttacoma/bonnie/pull/815
- https://github.com/projecttacoma/cql_qdm_patientapi/pull/65
- https://github.com/projectcypress/health-data-standards/pull/528

### CODE REVIEW

**Comment by lukoktonos** [comment]
The Travis CI test fails because the corresponding branch in hds is needed.  It passes locally with the gemfile pointing hds and cql_qdm_patientapi to the branch of the same name.

File: app/assets/javascripts/templates/patient_builder/edit_value.hbs [hunk 1]
```diff
@@ -46,58 +46,108 @@
     </div>
   </div>
 {{else}}
+  {{#ifCond type '==' 'FAC'}}
```
Comment (mayerm94) [inline app/assets/javascripts/templates/patient_builder/edit_value.hbs]: this should be en else if. If Cmp (CMP layout)..... else if FAC(Facility layout) ... else... (logic used for Code, TimeStamp... Scalar layouts). Indenting here seems weird too

Comment (lukoktonos) [reply app/assets/javascripts/templates/patient_builder/edit_value.hbs]: I wanted to use an elseif, but that's not supported in hbs, so I had to use an 
```
if
    (normal if logic goes here)
else
    if 
        (normal elseif logic goes here)
    else 
        (normal else logic goes here)
```
in order to achieve that effect.

**Comment by edeyoung** [comment]
Additional work required. Closing for now.

**Pull request closed without merging**